test/indexer/data/f0731ab4-6b80-4eed-97c9-4984de81a47c.metadata.json

```diff
@@ -190,7 +190,14 @@
         "describedBy": "https://schema.humancellatlas.org/type/file/6.0.0/analysis_file",
         "file_core": {
             "file_name": "merged-cell-metrics.csv.gz",
-            "format": "unknown"
+            "format": "unknown",
+            "content_description": [
```

Comment:
Is this realistic? Did DCP/2 Analysis provide this field.

Reply:
This `content_description` value was copied from another canned bundle. I checked a couple snapshots and found a more realistic value `Gene expression matrix` (data:3112) to associate with the `csv.gz` analysis file.

I wasn't able to find any snapshots containing a `.npz` analysis file with `content_description` set, so I also removed the `content_description` I had added to the file `sparse_counts.npz` and instead added it to the other `csv.gz` analysis file in the bundle so the test cases would have 2 analysis file & 1 supplementary file matrices to test with.

Reply:
But to confirm. Do the analysis subgraphs set the content_description on the actual loom files? Or is this a organic CGM subgraph?